src/meta/model_v2/src/hummock_version_delta.rs

```diff
@@ -32,6 +32,7 @@ pub struct Model {
     pub safe_epoch: Epoch,
     pub trivial_move: bool,
     pub gc_object_ids: SstableObjectIds,
+    pub new_table_watermarks: NewTableWatermarks,
```

Comment:
Need add new column in migration script
https://github.com/risingwavelabs/risingwave/blob/main/src/meta/model_v2/migration/src/m20231008_020431_hummock.rs